sys/cbor/cbor.c

```diff
@@ -236,12 +233,12 @@ void dump_memory(const unsigned char *data, size_t size)
         return;
     }
 
-    DEBUG("0x");
+    printf("0x");
```

Comment:
I forgot to update my last comment. The function is also called if an assert failes to show the difference. [CBOR_CHECK_SERIALIZED](https://github.com/OTAkeys/RIOT/blob/c180e667a9da10c0605a480d361b29290ae6f4bb/tests/unittests/tests-cbor/tests-cbor.c#L37) so the function isn't removed